<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBCreationWindow</class>
 <widget class="QWidget" name="DBCreationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="b_choose_src">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Sourse dir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_choose_dst">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Destination dir</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="s_average_count">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="l_average_count">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Average faces count</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="l_image_count">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Max image count</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="s_image_count">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="value">
    <number>4</number>
   </property>
  </widget>
  <widget class="QPushButton" name="b_create_db">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Create DB</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_src">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>351</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>/home/siobhan/UJ/Masters_stuff/db/src</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_dst">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>351</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>/home/siobhan/UJ/Masters_stuff/db/dst</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_info">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
